Userland/Libraries/LibVideo/MatroskaDocument.h

```diff
@@ -122,8 +122,8 @@ class Block {
     bool discardable() const { return m_discardable; }
     void set_discardable(bool discardable) { m_discardable = discardable; }
     u64 frame_count() const { return m_frames.size(); }
-    const ByteBuffer& frame(size_t index) const { return m_frames.at(index); }
-    void add_frame(const ByteBuffer& frame) { m_frames.append(move(frame)); }
+    ByteBuffer const& frame(size_t index) const { return m_frames.at(index); }
+    void add_frame(ByteBuffer const& frame) { m_frames.append(frame); }
```

Comment:
I think making this `const` leads to an additional copy, as you always create a new `ByteBuffer` with `ByteBuffer::copy` at the call sites anyway. So this should probably take the `ByteBuffer` by value (instead of non-`const` reference) and use `move`.
```suggestion
    void add_frame(ByteBuffer frame) { m_frames.append(move(frame)); }
```